training/models/training_material.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+
+from openerp import fields, models
+from .common import M
+
+
+class MaterialType(models.Model):
+    """Type of the training materials."""
+    _name = M % "material_type"
+    _sql_constraints = [("unique_name",
+                         "UNIQUE(name)",
+                         "Name must be unique.")]
+
+    name = fields.Char(required=True, index=True, translate=True)
+    material_ids = fields.One2many(
+        M % "material",
+        "type_id",
+        string="Materials",
+        help="Available materials of this type.")
+
+
+class Material(models.Model):
+    """Training material delivered to students."""
+    _name = M % "material"
```

Comment:
Yeah, it's a pity to not use product for this purpose. There's no problem with deliveries and so on. You can control your flow with for example invoice_control field to say if it's invoiceable or not.